products/memgraph/v0.15.0/reference_guide/other-features.md

```diff
@@ -159,3 +159,150 @@ to true is matched:
 MATCH (n)
 RETURN CASE WHEN n.height < 30 THEN "short" WHEN n.height > 300 THEN "tall" END
 ```
+
+### Inspecting queries
+
+Before a Cypher query is executed, it is converted into an internal form
+suitable for execution, known as a *plan*. A plan is a tree-like data structure
+describing a pipeline of operations which will be performed on the database in
+order to yield the results for a given query. Every node within a *plan* is
```

Comment:
I wouldn't put "plan" in italic here because you introduced the term two sentences ago.

Reply:
Good catch, I forgot to remove that.